raiden/tests/api/test_api.py

```diff
@@ -799,3 +799,25 @@ def test_connect_token_network(
     assert channels[0]['balance'] == expected_balance
     assert channels[1]['balance'] == expected_balance
     assert channels[2]['balance'] == expected_balance
+    assert channels[0]['state'] == 'opened'
```

Comment:
you should use the constants like `raiden.transfer.state.CHANNEL_STATE_OPENED` to compare the state